### ISSUE TITLE
Plugin grafana-zabbix missing 

### ISSUE THREAD

**Issue by Igor-Lopes** [alexanderzobnin/grafana-zabbix#178]
I've tried to install the plugin from the release package and downloaded the last realease, but "zabbix-plugin", as stated on the documentation and installation instructions, is not there after unpacking the tar  file

**Comment by alexanderzobnin** [comment]
Zabbix plugin will be available via grafana-cli soon. What kind of error with installation from tarball?

**Comment by Igor-Lopes** [comment]
I'm following the instructions from the documentation, so after unpacking the file I'm copying it to /var/lib/grafana/plugins, which is the correct repository for grafana 3.0. I restarted he grafana-server service, however the plugin does not work, since it's not possible to select zabbix when creating a new data source. I might be doing something wrong or this is just a unexpected behavior. Thank you for your help !

**Comment by alexanderzobnin** [comment]
Sorry, I need add some explamation to docs. You need to go to plugins in side menu, then select apps -> zabbix -> config and enable zabbix app. Then you cann add a datasource.

**Comment by Igor-Lopes** [comment]
Okay, I'll try it when I have access to the server. So am I installing the plugin correctly, by unpacking the tar file and copying it to /var/lib/grafana/plugins and restarting the service? 
Thank you very much for your help !

**Comment by alexanderzobnin** [comment]
Yes. You can also check grafana log. If plugin was founded, you will see it in log.

**Comment by Igor-Lopes** [comment]
The plugin worked just fine, after following the instructions you provided. However, I noticed that the last stable version for gafana is 2.6 and there is no compatible plugin for this version in the releases page for the plugin( I've tested them). The user has to decide between a beta and unstable version of the plugin and application or use an older stable version to work with the plugin for the versions 2.5.X. Running a beta and unstable application or an older version of the application in a production server is not good at all. I would like to suggest you to provide a stable version of the grafana-zabbix plugin for grafana 2.6. I'll be happy to help you with testing.
